src/amo/pages/Block/index.js

```diff
@@ -71,8 +71,10 @@ export class BlockBase extends React.Component<InternalProps> {
       return null;
     }
 
+    const { __html = '' } = sanitizeHTML(block.reason);
+
     return (
-      <p className="Block-reason">{block ? block.reason : <LoadingText />}</p>
+      <p className="Block-reason">{block ? __html : <LoadingText />}</p>
```

Comment:
I think you can inline the call to `sanitizeHTML` here:

```suggestion
      <p className="Block-reason">{block ? sanitizeHTML(block.reason).__html : <LoadingText />}</p>
```

Let me know what you think.

Reply:
Thanks. I made another commit to make the call inline.